ci-operator/step-registry/gcp/provision/check-osdisk-encryption/gcp-provision-check-osdisk-encryption-ref.metadata.json

```diff
@@ -0,0 +1,10 @@
+{
+	"path": "gcp/provision/check-osdisk-encryption/gcp-provision-check-osdisk-encryption-ref.yaml",
```

Comment:
How about moving the folder to `ci-operator/step-registry/cucushift/installer/check/gcp/disk-encryption` for the post check.